### PR TITLE
Adds tests for modis client.

### PR DESCRIPTION
**Pull request by Raahul-Singh**

### CODE REVIEW

**Comment by kakirastern** [comment]
The CI checks are not yet passing though...

File: wildfirepy/net/modis/__init__.py [hunk 1]
```diff
@@ -1 +1 @@
-from wildfirepy.net.modis.downloader import ModisBurntDownloader
+from .downloader import *
```
Comment (masterashu) [inline wildfirepy/net/modis/__init__.py:1]: Use absolute paths while importing.
```suggestion
from wildfirepy.net.modis.downloader import *
```

File: wildfirepy/net/util/__init__.py [hunk 2]
```diff
@@ -1 +1 @@
-from wildfirepy.net.util.base import ModisHtmlParser, URLOpenerWithRedirect
+from .modis import *
```
Comment (masterashu) [inline wildfirepy/net/util/__init__.py:1]: Use absolute paths while importing.

```suggestion
from wildfirepy.net.util.modis import *
```

File: wildfirepy/net/tests/test_modis.py [hunk 3]
```diff
@@ -0,0 +1,82 @@
+import pytest
```
Comment (masterashu) [inline wildfirepy/net/tests/test_modis.py:1]: We should use a separate directory at the root level for all the tests.

Comment (Raahul-Singh) [reply wildfirepy/net/tests/test_modis.py:1]: We'll add one for project-level tests. These are local module level tests.

File: wildfirepy/net/__init__.py [hunk 4]
```diff
@@ -1,2 +0,0 @@
-from wildfirepy.net import util as _
```
Comment (masterashu) [inline wildfirepy/net/__init__.py:1]: It should be the following 
```python
from wildfirepy.net import util
from wildfirepy.net import modis
```
other wise the imports such as 
> ```python
> from wildfirepy import net
> x = net.modis.ClassOrMethod
> ```

will not work.

**Comment by Raahul-Singh** [comment]
Thanks, @masterashu!

**Pull request merged**